src/toil/resource.py

```diff
@@ -378,7 +378,8 @@ def forModule(cls, name):
                 dirPathTail = filePath.pop()
                 assert dirPathTail == package
             dirPath = os.path.sep.join(filePath)
-        assert os.path.isdir(dirPath)
+        require(os.path.isdir(dirPath),
+                'Bad directory path %s for module %s. Maybe it contains egg-links?', dirPath, name)
```

Comment:
right, that's why the error warns them that maybe they are using egg-links in their directory. If this is not clear I can change it.